docs/entity-design.md

```
@@ -0,0 +1,321 @@
+
+OpenFGA's ability to model and evaluate granular, per-object access business requirements makes our permissions more transparent and auditable. In addition, supporting direct, context-aware access to individual objects in LFX opens the door to supporting AI use-cases like MCP servers for managing committees, meetings, and more.
+
+### Core concepts
+
+Understanding the following core concepts will help you navigate access control in the LFX v2 platform.
+
+**Model**: The schema that defines the types of entities and the kinds of relations users and objects can have with each other (e.g., users can be members of teams, teams can own projects, projects have a parent project). Our model is version-controlled and stored in this repository.
+
+**Tuple**: A relation between two _specific_ objects of any type, conforming to the model, which is stored as a fact in OpenFGA. For example: "user:alice is a member of team:frontend" or "project:platform is the parent of project:web-app". The v2 platform maintains a live sync that indexes LFX data (projects, committees, etc.) into corresponding OpenFGA tuples, in real time.
+
+**Relationship**: A relationship is the traversal or chaining of tuples, as defined by the model, to determine if a given *transitive* relationship exists. For example, if Alice is a member of the frontend team, and the frontend team owns the web-app project, then Alice has an ownership relationship to the web-app project. Throughout this document, when we refer to a "relationship", we always an queried relationship derived from chained tuples and the model definitions, and we will use the term "tuple" when referring to a *direct* relation "fact". The v2 platform uses OpenFGA's "batch check" endpoint to query relationships.
```

markdownlint-cli2 (0.21.0)

[warning] Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)